eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/deprecated/DeprecatedVariantMongoReader.java

```diff
@@ -107,18 +107,16 @@ protected List<Variant> getVariants(Document deprecatedVariant) {
         Document inactiveEntity = inactiveObjects.iterator().next();
         String contig = inactiveEntity.getString(VariantMongoAggregationReader.CONTIG_FIELD);
         long start = inactiveEntity.getLong(VariantMongoAggregationReader.START_FIELD);
-        String type = inactiveEntity.getString(TYPE_FIELD);
 
         for (Document submittedVariantOperation : submittedVariantOperations) {
             Collection<Document> inactiveEntitySubmittedVariant = (Collection<Document>) submittedVariantOperation
                     .get("inactiveObjects");
             Document submittedVariant = inactiveEntitySubmittedVariant.iterator().next();
-            long submittedVariantStart = submittedVariant.getLong(START_FIELD);
             String submittedVariantContig = submittedVariant.getString(CONTIG_FIELD);
             String reference = submittedVariant.getString("ref");
             String alternate = submittedVariant.getString("alt");
 
-            if (isSameLocation(contig, start, submittedVariantContig, submittedVariantStart, type)) {
+            if (contig.equals(submittedVariantContig)) {
```

Comment:
removed the condition for checking the start,  but was not sure if we needed to remove the contig checking as well, so left it for now

Reply:
I think we need to also remove the contig condition. 
It does not make sense to check the contig if we don't check the start.

Reply:
removed